app/services/idv/analytics_events_enhancer.rb

```diff
@@ -1,56 +1,58 @@
 module Idv
   module AnalyticsEventsEnhancer
-    DECORATED_METHODS = %i[
-      idv_cancellation_confirmed
-      idv_cancellation_go_back
-      idv_cancellation_visited
-      idv_forgot_password
-      idv_forgot_password_confirmed
-      idv_final
-      idv_gpo_address_letter_enqueued
-      idv_gpo_address_letter_requested
-      idv_in_person_ready_to_verify_visit
-      idv_letter_enqueued_visit
-      idv_personal_key_acknowledgment_toggled
-      idv_personal_key_downloaded
-      idv_personal_key_submitted
-      idv_personal_key_visited
-      idv_phone_confirmation_form_submitted
-      idv_phone_confirmation_otp_rate_limit_attempts
-      idv_phone_confirmation_otp_rate_limit_locked_out
-      idv_phone_confirmation_otp_rate_limit_sends
-      idv_phone_confirmation_otp_resent
-      idv_phone_confirmation_otp_sent
-      idv_phone_confirmation_otp_submitted
-      idv_phone_confirmation_otp_visit
-      idv_phone_confirmation_vendor_submitted
-      idv_phone_error_visited
-      idv_phone_of_record_visited
-      idv_phone_otp_delivery_selection_visit
-      idv_phone_otp_delivery_selection_submitted
-      idv_proofing_resolution_result_missing
-      idv_enter_password_submitted
-      idv_enter_password_visited
-      idv_please_call_visited
-      idv_start_over
+    IGNORED_METHODS = %i[
+      idv_acuant_sdk_loaded
+      idv_back_image_added
+      idv_back_image_clicked
+      idv_barcode_warning_continue_clicked
+      idv_barcode_warning_retake_photos_clicked
+      idv_capture_troubleshooting_dismissed
+      idv_consent_checkbox_toggled
+      idv_exit_optional_questions
+      idv_front_image_added
+      idv_front_image_clicked
+      idv_image_capture_failed
+      idv_in_person_ready_to_verify_sp_link_clicked
+      idv_in_person_ready_to_verify_what_to_bring_link_clicked
+      idv_in_person_switch_back_submitted
+      idv_in_person_switch_back_visited
+      idv_link_sent_capture_doc_polling_complete
+      idv_link_sent_capture_doc_polling_started
+      idv_mobile_device_and_camera_check
+      idv_native_camera_forced
+      idv_verify_in_person_troubleshooting_option_clicked
+      idv_warning_action_triggered
+      idv_warning_shown
     ].freeze
```

Comment:
since we're doing a membership check only, we can slightly optimize by saving as a set?
```suggestion
      idv_warning_shown
    ].to_set.freeze
```